### ISSUE TITLE
BBB 2.3 Mic not working externally

### ISSUE THREAD

**Issue by markginter31** [bigbluebutton/bigbluebutton#10197]
Everything works on the internal network (using the public fqdn) -- but if anyone outside the local network tries to connect the microphone - it fails.  But only the mic.  Listening only works.  Video camera works.  Only the mic throws a 1007 error.  Anyone see something similar?

Also - the moment I edit the sip.properties file and use the external IP - the echo test fails.  If I reassign the internal IP then everything works internally, and everything but the mic works externally. 

I do have a BBB 2.2 setup in a virtual machine that works.  It's on ubuntu16.04 and I noticed that I can run:
stun 172.217.212.127:19302
STUN client version 0.97
Primary: Open
Return value is 0x000001

whereas my 2.3 returns:
stun 172.217.212.127:19302
STUN client version 0.97
Primary: Independent Mapping, Independent Filter, preserves ports, no hairpin
Return value is 0x000013

I'm thinking my problem lies in Ubuntu 18.04's ability to allow this connection.  It gives the same message whether or not I have the firewall enabled on the 18.04 machine.

**Comment by markginter31** [comment]
So - I noticed that on 16.04 there was a loopback interface that had my public(external) IP attached to it.  I decided to use netplan on 18.04 to create a loopback interface identical to the one on 16.04.  Then I made sure to edit the freeswitch files to match my 16.04 setup.  Now it works.  I don't know why -- it's not clear why my external IP got attached to a loopback interface in 16.04 but not in 18.04.
